.travis.yml

```diff
@@ -12,6 +12,9 @@ jobs:
         - lint
       git:
         submodules: false # avoid cloning ethereum/tests
+      cache:
+        directories:
+          - ./build/cache/
```

Comment:
If you keep the version numbers in the file names, the cache will forever grow, downloading all past Go releases that we've ever used. That's why I placed it in go.tar.gz and not go1.13.4.tar.gz. Same for the other linter bundles.

Reply:
Hmm, I don't like this solution. We need the version number to verify the download hash in a sane way that doesn't involve passing in on the command line.

Reply:
I checked now and Travis explicitly says you're not supposed to cache downloads. So removing the cache is an option. I do think it's faster and more reliable though.